raiden_libs/messages/json_schema.py

```diff
@@ -145,6 +145,54 @@ def make_properties_required(schema):
     }
 }
 
+PATHS_REPLY_SCHEMA = {
+    'type': 'object',
+    'required': [
+        'token_network_address',
+        'target_address',
+        'value',
+        'num_paths',
+        'chain_id',
+        'nonce',
+        'paths',
+        'estimated_fees',
+        'signature',
+    ],
+    'properties': {
+        'token_network_address': {
+            'type': 'string',
+        },
+        'target_address': {
+            'type': 'string',
+        },
+        'value': {
+            'type': 'integer',
+            'minimum': 1
+        },
+        'num_paths': {
+            'type': 'integer',
+            'minimum': 1
+        },
+        'chain_id': {
+            'type': 'integer',
+            'minimum': 1
+        },
+        'nonce': {
+            'type': 'integer',
+            'minimum': 0
+        },
+        'paths': {
+            'type': 'array',
+        },
+        'estimated_fees': {
```

Comment:
I'd prefer to put them in the same array. I imagine this as a list of paths objects, each containing a list of nodes and a calculated cost.

Reply:
Agreed.